### PR TITLE
SEO-174654-JavaScript-Image-With-anchor-text

### PR DESCRIPTION
**Pull request by Omanyadick**
Hi @Aishwarya-Ganesan, Kindly review my PR below.
Title | Description
-- | --
|Task Category | Site Audit work Links with no anchor text|
|Content Task Link/Mail Screenshot | NA|
|Hotfix | Hotfix- https://github.com/syncfusion-content/javascript-docs/pull/504|
|Ticket/Task link | https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/187693/|
|Work link | https://syncfusion.sharepoint.com/:x:/r/sites/SEOWebsiteAuditing/_layouts/15/Doc.aspx?sourcedoc=%7BFFBBFCD9-0FB1-48D9-8F89-2C6BE58849E3%7D&file=Dickson.xlsx&action=default&mobileredirect=true|

Changes Made | Reason for change | Reason for not identifying earlier
-- | -- | --
 Links with no anchor text| Aesthetic and Professional Appearance to users. |  

Regards
Dickson J

### CODE REVIEW

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: js/PdfViewer/Getting-Started.md [hunk 1]
```diff
@@ -17,13 +17,13 @@ This section explains how to create the PDF viewer in your web application using
 
 Create a new project in the Visual Studio by selecting the ASP.NET Empty Web Application template. The following screenshot displays the Project Creation Wizard in Visual Studio 2012.
 
-![GettingStarted](Getting-Started_images/Getting-Started_img1.jpeg)
+![Getting Started with PDF Viewer.](Getting-Started_images/getting-started-with-pdf-viewer.jpeg)
```
[the remark this answers is not in the document]
Comment (Omanyadick) [reply js/PdfViewer/Getting-Started.md:20]: Resolved

File: js/PdfViewer/Getting-Started.md [hunk 2]
```diff
@@ -17,13 +17,13 @@ This section explains how to create the PDF viewer in your web application using
...
 
 **Create HTML Page**
 
 Right-click the project and select New Item option from the ‘Add’ menu to add a HTML page to the application. Name the page as **Default.html** and click OK.
 
-![GettingStarted](Getting-Started_images/Getting-Started_img2.jpeg)
+![Getting Started with default HTML.](Getting-Started_images/getting-started-with-default-html.jpeg)
```
[the remark this answers is not in the document]
Comment (Omanyadick) [reply js/PdfViewer/Getting-Started.md:26]: Checked

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: js/PdfViewer/Getting-Started.md [hunk 3]
```diff
@@ -17,13 +17,13 @@ This section explains how to create the PDF viewer in your web application using
 
 Create a new project in the Visual Studio by selecting the ASP.NET Empty Web Application template. The following screenshot displays the Project Creation Wizard in Visual Studio 2012.
 
-![GettingStarted](Getting-Started_images/Getting-Started_img1.jpeg)
+![Creation Wizard in JavaScript PDF Viewer.](getting-started_images/creation-wizard-in-javascript-pdf-viewer.jpeg)
```
[the remark this answers is not in the document]
Comment (Omanyadick) [reply js/PdfViewer/Getting-Started.md:20]: correct casing used.

File: js/PdfViewer/Getting-Started.md [hunk 4]
```diff
@@ -519,7 +519,7 @@ Run the sample application and you can see the PDF Viewer on the page as display
 
 <https://www.syncfusion.com/downloads/support/directtrac/general/ze/PdfViewer_GettingStarted-67533367>
 
-![GettingStarted](Getting-Started_images/Getting-Started_img5.png)
+![Sample application in JavaScript PDF Viewer.](getting-started_images/sample-application-in-javascript-pdf-viewer.png)
```
Comment (Yvone-Atieno) [inline js/PdfViewer/Getting-Started.md:522]: Check image Alt

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**